### ISSUE TITLE
2.0.4 Error on linux

### ISSUE THREAD

**Issue by Rigel772** [digimezzo/vitomu#28]
on npm start:

```npm start             

> Vitomu@2.0.4 start /media/storage/oskar/Apps/vitomu
> npm run postinstall:electron && npm-run-all -p ng:serve electron:serve


> Vitomu@2.0.4 postinstall:electron /media/storage/oskar/Apps/vitomu
> node postinstall

[Error: ENOENT: no such file or directory, open 'node_modules/@angular-devkit/build-angular/src/angular-cli-files/models/webpack-configs/browser.js'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: 'node_modules/@angular-devkit/build-angular/src/angular-cli-files/models/webpack-configs/browser.js'                                                                                          
}

> Vitomu@2.0.4 electron:serve /media/storage/oskar/Apps/vitomu
> wait-on http-get://localhost:4200/ && npm run electron:serve-tsc && electron . --serve


> Vitomu@2.0.4 ng:serve /media/storage/oskar/Apps/vitomu
> ng serve

An unhandled exception occurred: Job name "..getProjectMetadata" does not exist.
See "/tmp/ng-1psKNw/angular-errors.log" for further details.
npm ERR! code ELIFECYCLE
npm ERR! syscall spawn
npm ERR! file sh
npm ERR! errno ENOENT
npm ERR! Vitomu@2.0.4 ng:serve: `ng serve`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the Vitomu@2.0.4 ng:serve script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/oskar/.npm/_logs/2020-11-23T12_07_39_157Z-debug.log
ERROR: "ng:serve" exited with 1.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! Vitomu@2.0.4 start: `npm run postinstall:electron && npm-run-all -p ng:serve electron:serve`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the Vitomu@2.0.4 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

**Comment by Rigel772** [comment]
from npm log:
```
0 info it worked if it ends with ok
1 verbose cli [
1 verbose cli   '/usr/bin/node',
1 verbose cli   '/usr/lib/node_modules/npm/bin/npm-cli.js',
1 verbose cli   'run',
1 verbose cli   'ng:serve'
1 verbose cli ]
2 info using npm@6.14.8
3 info using node@v15.2.1
4 verbose run-script [ 'preng:serve', 'ng:serve', 'postng:serve' ]
5 info lifecycle Vitomu@2.0.4~preng:serve: Vitomu@2.0.4
6 info lifecycle Vitomu@2.0.4~ng:serve: Vitomu@2.0.4
7 verbose lifecycle Vitomu@2.0.4~ng:serve: unsafe-perm in lifecycle true
8 verbose lifecycle Vitomu@2.0.4~ng:serve: PATH: /usr/lib/node_modules/npm/node_modules/npm-lifecycle/node-gyp-bin:/media/storage/oskar/Apps/vitomu/node_modules/.bin:/usr/lib/node_modules/npm/node_modules/npm-lifecycle/node-gyp-bin:/media/storage/oskar/Apps/vitomu/node_modules/.bin:/home/oskar/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl:/var/lib/snapd/snap/bin
9 verbose lifecycle Vitomu@2.0.4~ng:serve: CWD: /media/storage/oskar/Apps/vitomu
10 silly lifecycle Vitomu@2.0.4~ng:serve: Args: [ '-c', 'ng serve' ]
11 info lifecycle Vitomu@2.0.4~ng:serve: Failed to exec ng:serve script
12 verbose stack Error: Vitomu@2.0.4 ng:serve: `ng serve`
12 verbose stack spawn ENOENT
12 verbose stack     at ChildProcess.<anonymous> (/usr/lib/node_modules/npm/node_modules/npm-lifecycle/lib/spawn.js:48:18)
12 verbose stack     at ChildProcess.emit (node:events:329:20)
12 verbose stack     at maybeClose (node:internal/child_process:1055:16)
12 verbose stack     at Process.ChildProcess._handle.onexit (node:internal/child_process:288:5)
13 verbose pkgid Vitomu@2.0.4
14 verbose cwd /media/storage/oskar/Apps/vitomu
15 verbose Linux 5.8.18-1-MANJARO
16 verbose argv "/usr/bin/node" "/usr/lib/node_modules/npm/bin/npm-cli.js" "run" "ng:serve"
17 verbose node v15.2.1
18 verbose npm  v6.14.8
19 error code ELIFECYCLE
20 error syscall spawn
21 error file sh
22 error errno ENOENT
23 error Vitomu@2.0.4 ng:serve: `ng serve`
23 error spawn ENOENT
24 error Failed at the Vitomu@2.0.4 ng:serve script.
24 error This is probably not a problem with npm. There is likely additional logging output above.
25 verbose exit [ 1, true ]
```

**Comment by digimezzo** [comment]
@Rigel772 did you perform **npm install** first?

**Comment by Rigel772** [comment]
Yes I have.

**Comment by digimezzo** [comment]
@Rigel772  I honestly don't know what is going on. You seem to be using Manjaro. I always use Manjaro to do my Linux tests. So this should work. It's like node_modules cannot be accessed.

**Comment by Rigel772** [comment]
I run it from home directory... should it make any difference?
Maybe there is something more missing from angular?
I do not have this folder after npm install:
`node_modules/@angular-devkit/build-angular/src/angular-cli-files/`

**Comment by digimezzo** [comment]
@Rigel772 You should run npm start from the Directory containing the project. node_modules must be a direct subfolder of the directory in which you run npm start.

**Comment by Rigel772** [comment]
that is what I do...

**Comment by digimezzo** [comment]
Fixed in commit 830ca156731b79ad511d543f02736d740de0d4e7
